project/project.sbt

```diff
@@ -2,7 +2,7 @@ import java.util.Locale
 
 libraryDependencies += "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
 
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.6")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
Not needed. `sbt-pgp` is a dependency of `sbt-web-build-base`'s *build*, it's not a library dependency of the plugin. `sbt-web-build-base` is still using sbt 0.13.16 to build itself, not sbt 1.0, hence it doesn't need any sbt 1.0 compatible plugins.